page_analyzer/utils/db.py

```diff
@@ -0,0 +1,90 @@
+import os
+from typing import Any, Dict, List
+
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def connect_db() -> Any:
+    conn = psycopg2.connect(os.getenv("DATABASE_URL"))
+    return conn
+
+
+def get_url_context(url_id: int) -> Dict[str, str]:
+    conn = connect_db()
+    cur = conn.cursor()
```

Comment:
I have a feeling that [dict-like cursor](https://www.psycopg.org/docs/extras.html#dictionary-like-cursor) could be useful for easier property access, wdyt?